odoo/tools/misc.py

```diff
@@ -1830,6 +1830,9 @@ def limited_field_access_token(record, field_name, timestamp=None):
     :type record: class:`odoo.models.Model`
     :param field_name: the field name of record to generate the token for
     :type field_name: str
+    :param scope: scope of the authentication, to have different signature for the same
+        message in different usage.
```

Comment:
I'm not sure if "message" is the right word here. It's more about record/field combination.